lib/undercover/result.rb

```diff
@@ -33,7 +33,7 @@ def uncovered?(line_no)
       end
 
       line_cov = coverage.find { |ln, _cov| ln == line_no }
-      line_cov && line_cov[1].zero?
+      line_cov && line_cov.last.zero?
```

Comment:
Instead of use the second position, which is the block number for branch coverage, I change to use the last value which is always the coverage number.